examples/blender/blender.js

```diff
@@ -87,8 +87,8 @@ async function main(subnetTag, driver, network) {
 
 program
   .option("--subnet-tag <subnet>", "set subnet name", "devnet-beta")
-  .option("--driver <driver>", "payment driver name, for example 'zksync'", "zksync")
-  .option("--network <network>", "network name, for example 'rinkeby'", "rinkeby")
+  .option("--driver <driver>", "payment driver name, for example 'erc20'")
```

Comment:
is the default not removed here altogether? ...

Reply:
Yes, it is.
@filipgolem you removed it in the previous version, was it intended?